tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "target": "es5",
-        "module": "esnext",
+        "module": "commonjs",
```

Comment:
how is this needed?

Reply:
This would be needed if we depend on dust from odin (in practice we're going the other way now) but I'm just going through and fixing this up everywhere. Without this I get a big screen of error message from jest